test/collections_test_helper.rb

```diff
@@ -0,0 +1,51 @@
+# rubocop:disable Metrics/LineLength
+require_relative "./test_helper"
+
+VALID_COLLECTION_METADATA_KEYS = %w[collection created_by display_name image items].freeze
+
+REQUIRED_COLLECTION_METADATA_KEYS = %w[display_name].freeze
```

Comment:
Seems like `items` and `collection` would also be required for a collection.

Reply:
I don't believe Collections are required to have items. I will look at adding collection.

Reply:
What is the point of an empty collection? Seems like at that point, all you have is the name and description, making it essentially a blog post.

Reply:
I am not sure.  @brandonrosage  Should we make items required?

Reply:
@damaneice We should make them required. There won't be a usable instance of a Collection without them. They'd just be confusing to users.